docs/business-plan/tools/build_hwp_tables_docx.py

```
@@ -0,0 +1,483 @@
+    )
+    add_page_break(document)
+
+    add_compact_flow(
+        document,
+        "그림 6. 현재 구현 상태와 다음 단계",
+        [
+            ("완료", "공통 데이터 구조\n온보딩·마이허브\nOCR 미리보기"),
+            ("데모 구현", "진행 캘린더\n기관 지도\n반응형 흐름"),
+            ("구현 예정", "규칙 계산 엔진\n다국어·근거 검색\n위험 상황 라우팅"),
+        ],
+        "1차 실증: 이주노동자 × E-7-4R 체류자격 트래커",
```

Ruff (0.16.2)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)